users.js

```diff
@@ -397,19 +423,16 @@ class User {
 		this.send(`|popup|` + message.replace(/\n/g, '||'));
 	}
 	getIdentity(roomid) {
-		if (this.locked) {
-			return '‽' + this.name;
-		}
-		if (this.namelocked) {
-			return '‽' + this.name;
+		if (this.locked || this.namelocked) {
+			return (Config.punishgroups.locked || '‽') + this.name;
```

Comment:
So, this is going to return `[Object object]shen` instead of `'‽snen` as we'd want, right?